locale/en.po

```diff
@@ -7268,6 +7238,36 @@ msgstr ""
 msgid "WIKI_NITROGENASE_UPGRADES"
 msgstr "No upgrades."
 
+msgid "WIKI_NITROPLAST_EFFECTS"
```

Comment:
There's a big problem here, the wiki content is automatically updated with a script from our wiki, and I don't see these edits on the wiki (at least there isn't a page called "nitroplast").

So could you make these edits on the wiki? I'll help out by moving the existing page to be named nitroplast you can then edit: https://wiki.revolutionarygamesstudio.com/wiki/Nitroplast

Reply:
Do these items only need to be updated on the wiki, or should the changes be done both here and on the wiki?

Reply:
It didn't seem to make any changes to my local files - at least nothing showed in VSCode.

Reply:
Has the wiki website been actually updated now?

Reply:
Yes, I updated the wiki page for Nitroplast.

Reply:
It looks like the script has now worked as the changes look correct to me now.